.github/workflows/ci.yml

```diff
@@ -56,3 +69,63 @@ jobs:
             -
                 name: "Run friendsofphp/php-cs-fixer"
                 run: "vendor/bin/php-cs-fixer fix --dry-run --diff"
+
+    test:
+        name: "Test PHP ${{ matrix.php-version }} + maker-bundle version: ${{ matrix.maker-test-version }}"
+
+        runs-on: ubuntu-latest
+
+        continue-on-error: ${{ matrix.allow-failures }}
+
+        strategy:
+            matrix:
+                php-version:
+                    - '7.1.33'
+                    - '7.3'
+                    - '7.4'
+                maker-test-version:
+                    - 'stable'
+                allow-failures: [false]
+                include:
+                    - php-version: '7.4'
+                      maker-test-version: 'dev'
+                      allow-failures: true
+
+        steps:
+            - name: Checkout
+              uses: actions/checkout@v2.3.3
+
+            - name: Install PHP with extensions
+              uses: shivammathur/setup-php@2.7.0
+              with:
+                  coverage: "none"
+                  php-version: ${{ matrix.php-version }}
+                  tools: composer:v2
+
+            - name: Add PHPUnit matcher
+              run: echo "::add-matcher::${{ runner.tool_cache }}/phpunit.json"
+
+            - name: Set Composer cache directory
+              id: composer-cache
+              run: echo "::set-output name=dir::$(composer config cache-files-dir)"
+
+            - name: Cache Composer
+              uses: actions/cache@v2.1.2
+              with:
+                  path: ${{ steps.composer-cache.outputs.dir }}
+                  key: ${{ runner.os }}-${{ matrix.php-version }}-composer-${{ matrix.dependencies }}-${{ matrix.variant }}
+                  restore-keys: ${{ runner.os }}-${{ matrix.php-version }}-composer-
+
+            - name: Require symfony/flex
+              run: composer global require --no-progress --no-scripts --no-plugins symfony/flex dev-main
```

Comment:
~dev-main~

Reply:
You mean we should remove it entirely?

Reply:
This is currently trying to install a package called "dev-main". 

Let's remove that entirely. 

If you really really want it there, then it should be `symfony/flex:dev-main`

Reply:
This part is ported from `.travis.yaml` 🤔  and this steps does not fail, I think .this one is failing:
https://github.com/symfony/maker-bundle/pull/701/files#r512590650

Reply:
What is missing? The link (unfortunately) doesn't go anywhere for me.

But in general, @Nyholm is correct - what you have here is not the correct syntax for a version. I have no idea why it's not failing with a `dev-main package not found` type of error.

Reply:
Looks like everything is fine now